.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!-- This form is for bug reports and feature requests ONLY!
+Also make sure that you visit our User Guide at https://kubevirt.io/user-guide/
+-->
+
+**Is this a BUG REPORT or FEATURE REQUEST?**:
+
+> Uncomment only one, leave it on its own line:
+>
+> /kind bug
+> /kind enhancement
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+- CDI version (use `kubectl get deployments hyperconverged-cluster-operator -o yaml`):
```

Comment:
CDI version is irrelevant in our case.
If it was deployed via OLM, it should be:
```suggestion
- HCO version (use `oc get csv -n kubevirt-hyperconverged`):
```
(notice the change from `kubectl` to `oc`, since OLM available in openshift.
And if it was deployed without OLM
In such case, the version is extracted in the following way:
```
HCO_IMAGE=`kubectl get deployment hco-operator -n kubevirt-hyperconverged -o=jsonpath='{.spec.template.spec.containers[0].image}'`
skopeo inspect docker://$HCO_IMAGE | jq '.Labels.version'
```